python/foglamp/services/south/ingest.py

```diff
@@ -457,6 +460,12 @@ def increment_discarded_readings(cls):
                 cls._discarded_readings_stats += readings
                 _LOGGER.exception('An error occurred while writing discarded statistics')
 
+            try:
+                await stats.add_update(cls._sensor_stats)
+                cls._sensor_stats = {}
+            except Exception:  # TODO catch real exception
```

Comment:
except Exception as ex:  and log with str(ex) ? or you mean to handle specific known exceptions stats.add_update can raise